rest/route/project.go

```diff
@@ -962,6 +923,147 @@ func (h *getProjectVersionsHandler) Run(ctx context.Context) gimlet.Responder {
 	return resp
 }
 
+// POST /rest/v2/projects/{project_id}/versions
+
+// versionsSetPriorityHandler is a RequestHandler for setting the priority of versions.
+type versionsSetPriorityHandler struct {
+	projectId string
+	url       string
+	opts      dbModel.SetVersionsPriorityOptions
+}
+
+func makeSetPriorityProjectVersionsHandler(url string) gimlet.RouteHandler {
```

Comment:
Should rename this to something more general in case we want to support things besides priority (since the route name is general)